.eleventy.js

```diff
@@ -1,51 +1,47 @@
-/* global chunkArray, createPagination, getUniqueTags */
+/* global chunkArray, createPagination */
 
 const errorOverlay = require("eleventy-plugin-error-overlay");
 const pluginPWA = require("eleventy-plugin-pwa");
 const eleventyNavigation = require("@11ty/eleventy-navigation");
 const fs = require("fs");
 
 const dataFetcherAirtable = require("./src/utils/data-fetcher-airtable.js");
-const dataFetcherWp = require("./src/utils/data-fetcher-wp.js");
 const htmlMinifyTransform = require("./src/transforms/html-minify.js");
 const parseTransform = require("./src/transforms/parse.js");
 const dateFilter = require("./src/filters/date.js");
 const htmlSymbolFilter = require("./src/filters/html-symbol.js");
 const markdownFilter = require("./src/filters/markdown.js");
-const turndownFilter = require("./src/filters/turndown.js");
+const paginateFilter = require("./src/filters/paginate.js");
 const slugFilter = require("./src/filters/slug.js");
 const w3DateFilter = require("./src/filters/w3-date.js");
 const randomizeFilter = require("./src/filters/randomize.js");
 const expanderShortcode = require("./src/shortcodes/expander.js");
 const imageAndTextShortcode = require("./src/shortcodes/image-and-text.js");
 const youtubeShortcode = require("./src/shortcodes/youtube.js");
 
-// Slugs for pages that should be excluded as public pages from the WeCount website.
-const privatePageSlugs = ["views", "news"];
-
 require("./src/js/utils.js");
 
+/**
+ * Get an array of unique tags from a collection.
+ *
+ * @param {Object} collection An Eleventy collection defined via the collections API: https://www.11ty.dev/docs/collections/#advanced-custom-filtering-and-sorting
+ * @returns {Array}
+ */
+const getUsedTags = function(collection) {
+	const tagsSet = new Set();
```

Comment:
Sets can only contain unique values, so this is a way to avoid having to filter the array once all the tags are added: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set#value_equality

Reply:
Got it. Thanks.